app/PaymentDrivers/StripePaymentDriver.php

```diff
@@ -714,6 +728,60 @@ public function processWebhookRequest(PaymentWebhookRequest $request)
                     }
                 }
             }
+        } elseif ($request->type === "payment_method.automatically_updated"){
+            // Will notify customer on updated information
```

Comment:
Do we need to do anything here?

Reply:
Probably yes, but Stripe has no documentation about it. During testing I was unable to create such an event. 

Theoretically Stripe would inform us, that the bank connection has changed, which results in a new payment token...